pkg/router/template/router.go

```diff
@@ -222,18 +223,61 @@ func (r *templateRouter) EnableRateLimiter(interval int, handlerFunc ratelimiter
 	glog.V(2).Infof("Template router will coalesce reloads within %v seconds of each other", interval)
 }
 
+// When there is no default cert, cobble one together from
+// the mounted secret, if present
+func secretToPem(outName string) error {
```

Comment:
Also, it there a reason that this code isn't in `CertificateWriter` like the old method was?